AGENTS.md

```
@@ -0,0 +1,149 @@
+### Testing
+
+No test suite configured. When added:
+```bash
+pnpm test                    # Run all tests
+pnpm test -- path/to/test.mjs  # Run single test file
+```
+
+### Linting
+
+No linter configured. Recommended:
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)